net/get-started/docker/dotnet6/_index.md

```diff
@@ -0,0 +1,221 @@
+---
+title: How to run Aspose.PDF for .NET 6 in Docker
+linktitle: Using Aspose.PDF for .NET 6 in Docker
+type: docs
+weight: 10
+url: /net/docker/dotnet6/
+description: Integrate Aspose.PDF functionality into your application using Docker Linux or Windows containers
+lastmod: "2024-01-21"
+sitemap:
+    changefreq: "monthly"
+    priority: 0.5
+---
+
+## Prerequisites
+
+Following examples tested with:
+
+* Docker v.20.10.11 and Docker Desktop 4.3.2
+* Visual Studio 2022 Community Edition v.17.0.5
+* .NET 6 SDK is used in the example, provided below.
+* Aspose.PDF for .NET v.22.01
+
+## Create sample application for Docker Linux Container
+
+1. Launch Visual Studio 2022 and choose **ASP.NET Core Web App (Model-View-Controller)** template and press **Next**
+1. In **Configure your new project** window set desired project name and location and press **Next**
+1. In **Additional information** window choose **.NET 6.0 (Long-term support)** and enable Docker support. You can also set **Docker OS** to Linux if needed.
+1. Press **Create**
+1. Choose **Tools->Nuget Package Manager->Package Manager Console** and install **Aspose.PDF for .NET** (use command `Install-Package Aspose.PDF`)
+
+### Generate PDF document using ASP.NET Core Web App in Linux container
+
+We will use code from **Complex Example** in this app. For more detailed explanation, please follow [this link](/pdf/net/complex-pdf-example/).
+
+1. Create `images` folder in `wwwroot` folder and put image `logo.png`. You can use download this image from [here](/pdf/net/docker/logo.png)
+1. Replace code in `HomeController.cs` with following snippet (please note that you can have another namespace):
+
+The following code snippet also work with [Aspose.PDF.Drawing](/pdf/net/drawing/) library.
```

Comment:
The following code snippet also works with [Aspose.PDF.Drawing](/pdf/net/drawing/) library.